### DISCUSSION TITLE
Question about developing firecms

### DISCUSSION THREAD

**Discussion by KULTI1995** [firecmsco/firecms#422] [General]
Due to the fact that I am in the decision stage, regarding the use of this cms in projects. I have a question.

Will adding-editing schemas, directly in the cms be available in self-hosted version and MIT license?
https://firecms.co/assets/medias/collection_editor_preview-bfb5a082bc0d3f37e7830d75407dd0c2.mp4

**Comment by fgatti675** [comment]
Hi @KULTI1995 
There is not a decision on this yet.
But there is a change we will distribute the editor as a licensed package.
Unfortunately we are not making barely any money with sponsors and we need resources to continue development.

**Reply by KULTI1995** [reply]
It is a bit sad. Every product that Myth, which is barely created, soon has paid services. How could we not keep it just MIT and give everyone a good self hosted product.

**Reply by fgatti675** [reply]
They core is staying MIT, but again, we need to make some money if we want to be able to continue developing it

**Comment by KULTI1995** [comment]
I can see that also many types have not been released and are waiting for the paid version. pity.

**Reply by fgatti675** [reply]
What do you mean?
